ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -76,9 +76,10 @@ def _parse_data_model(self):
                     database = mnode["database"] if mnode["database"] else "default"
                     schema = mnode["schema"] if mnode["schema"] else "default"
                     raw_sql = mnode.get("raw_sql", "")
```

Comment:
@ulixius9 do we need to handle anything here?

Reply:
Didn't update this because this wasn't generating any activity log.